Documentation/ApiOverview/UpdateWizards/Creation.rst

```diff
@@ -53,7 +53,7 @@ methods::
         */
        public function getIdentifier(): string
        {
-         return 'exampleUpdateWizard';
+         return 'myExtension_exampleUpdateWizard';
```

Comment:
This must be `extName_exampleUpdateWizard` following the existing code.

Reply:
Pushed another commit. All points should be resolved.